src/components/accessories/patientTherapy/therapyForm/TherapyForm.tsx

```diff
@@ -41,7 +42,10 @@ const TherapyForm: FC<TherapyProps> = ({
         name: "endDate",
         message: t("therapy.validatelastdate"),
         test: function (value) {
-          return moment(value).isSameOrAfter(moment(this.parent.startDate));
+          if (moment(+value).isValid()) {
+            return moment(+value).isSameOrAfter(moment(+this.parent.startDate));
+          } else
+            return moment(value).isSameOrAfter(moment(this.parent.startDate));
```

Comment:
As below, if the date is invalid it shouldn't be formatted.

Reply:
OK